core/ordersync/ordersync.go

```diff
@@ -162,6 +162,8 @@ func New(ctx context.Context, node *p2p.Node, subprotocols []Subprotocol) *Servi
 		sids = append(sids, subp.Name())
 		supportedSubprotocols[subp.Name()] = subp
 	}
+	// TODO(jalextowle): We should ensure that there were no duplicates -- there
```

Comment:
@albrow I left this in intentionally to prompt a discussion about this. Do you agree that we should either (a) fail if there are duplicates or (b) filter them out?